pkg/argocd/update.go

```diff
@@ -621,10 +621,15 @@ func getWriteBackConfig(app *v1alpha1.Application, kubeClient *kube.KubernetesCl
 	return wbc, nil
 }
 
-func parseDefaultTarget(appName string, path string) string {
-	defaultTargetFile := fmt.Sprintf(common.DefaultTargetFilePattern, appName)
-
-	return filepath.Join(path, defaultTargetFile)
+func parseDefaultTarget(appNamespace string, appName string, path string, kubeClient *kube.KubernetesClient) string {
+    log.Warnf("Namspace : %v", kubeClient)
```

Comment:
Should this be info level, instead of warn? Can we also make it clear that this is the control plane namespace? For the arg, did you mean `kubeclient.Namespace`?